WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/calendars/OnCallCalendar.jsp

```diff
@@ -271,8 +271,8 @@
         <%
             if (startDate != null && endDate != null) {
         %>
-        startDateOfSchedule = new Date("<%= startDate %>");
-        endDateOfSchedule = new Date("<%= endDate %>");
+        startDateOfSchedule = new Date("<%= h(startDate) %>");
+        endDateOfSchedule = new Date("<%= h(endDate) %>");
```

Comment:
```suggestion
        endDateOfSchedule = new Date("<%= q(endDate) %>");
```

Reply:
Indeed, but q() handles adding the quotes on its own. Updated change pushed.